### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by seguler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,33 @@
+# Azure Storage Blob SDK for Go
+The Microsoft Azure Storage SDK for Go allows you to build applications that takes advantage of Azure's scalable cloud storage. 
 
-# Contributing
+This repository contains the open source Blob SDK for Go. Documentation and examples can be found [here](https://godoc.org/github.com/Azure/azure-storage-blob-go/2016-05-31/azblob).
 
+## Features
+* Blob Storage
+	* Create/Read/Update/Delete Block Blobs
```
Comment (JeffreyRichter) [inline README.md]: Do you want to add that you can Create/Delete containers too?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,33 @@
...
+	* Create/Read/Update/Delete Page Blobs
+	* Create/Read/Update/Delete Append Blobs
+
+## Getting Started
+* If you don't already have it, install [the Go Programming runtime](https://golang.org/dl/)
```
Comment (JeffreyRichter) [inline README.md]: Don't call this the "runtime" -- it's not the term they use. Say  "install the Go distribution" or just "install Go"

File: README.md [hunk 3]
```diff
@@ -1,6 +1,33 @@
...
+* Go get the SDK:
+
+```go get -u https://github.com/Azure/azure-storage-blob-go```
```
Comment (JeffreyRichter) [inline README.md]: This should be: go get github.com/Azure/azure-storage-blob-go/2016-05-31/azblob

File: README.md [hunk 4]
```diff
@@ -1,6 +1,33 @@
...
+		
+## SDK Architecture
+
+* Storage SDK for Go provides 2 set of APIs: high-level, and low-level APIs
```
Comment (JeffreyRichter) [inline README.md]: Prepend: "The Azure ..."

File: README.md [hunk 5]
```diff
@@ -1,6 +1,33 @@
...
+	* ServiceURI, ContainerURI and BlobURI objects provide the low-level API functionality and maps one-to-one to the [Azure Storage Blob REST APIs](https://docs.microsoft.com/en-us/rest/api/storageservices/blob-service-rest-api)
+	* A set of high-level APIs are provided in highlevel.go file. These functions provide high level abstractions for convenience like uploading a large stream to Blob storage using multiple PutBlock requests
```
Comment (JeffreyRichter) [inline README.md]: The high-level APIs provide high-level convenience abstractions such as uploading a large stream to a block blob (using multiple PutBlock requests).

File: README.md [hunk 6]
```diff
@@ -1,6 +1,33 @@
...
+
+## Code Samples
+* [Blob Storage Samples](https://github.com/seguler/azure-storage-blob-go/blob/master/2016-05-31/azblob/zt_examples_test.go)
```
Comment (JeffreyRichter) [inline README.md]: This is the link for examples: https://godoc.org/github.com/Azure/azure-storage-blob-go/2016-05-31/azblob#pkg-examples

File: README.md (hunk 1 above)
Comment (seguler) [reply README.md]: Done. Included listing operation as well

File: README.md (hunk 2 above)
Comment (seguler) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (seguler) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (seguler) [reply README.md]: Slight modification on this... I assume we will add more high-level APIs, so dropped 'a few' from there... Good ?

File: README.md (hunk 6 above)
Comment (seguler) [reply README.md]: Done

**Pull request merged**